internal/sdkv2provider/resource_cloudflare_address_map_test.go

```diff
@@ -0,0 +1,105 @@
+package sdkv2provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/cloudflare/cloudflare-go"
+	"github.com/cloudflare/terraform-provider-cloudflare/internal/consts"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccCloudflareAddressMap(t *testing.T) {
```

Comment:
just waiting to get this all hooked up on the acceptance testing account so these can run